include/rabbit.hrl

```diff
@@ -147,8 +147,10 @@
 -define(EMPTY_FRAME_SIZE, 8).
 
 -define(MAX_WAIT, 16#ffffffff).
--define(SUPERVISOR_WAIT, infinity).
--define(WORKER_WAIT, 30000).
+-define(SUPERVISOR_WAIT,
+        application:get_env(rabbit, supervisor_shutdown_timeout, infinity)).
```

Comment:
`application:get_env/3` is not available in OTP R16B03. This should use `rabbit_misc:get_env/3`.